src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/ManagingOfficerIndividualValidator.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.ManagingOfficerIndividualDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.CountryLists;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.DateValidators;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.StringValidators;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationUtils.getQualifiedFieldName;
+
+@Component
+public class ManagingOfficerIndividualValidator {
+
+    private final AddressDtoValidator addressDtoValidator;
+
+    @Autowired
+    public ManagingOfficerIndividualValidator(AddressDtoValidator addressDtoValidator) {
+        this.addressDtoValidator = addressDtoValidator;
+    }
+
+    public Errors validate(List<ManagingOfficerIndividualDto> managingOfficerIndividualDtoList, Errors errors, String loggingContext) {
+        for (ManagingOfficerIndividualDto managingOfficerIndividualDto : managingOfficerIndividualDtoList) {
+            validateFirstName(managingOfficerIndividualDto.getFirstName(), errors, loggingContext);
+            validateLastName(managingOfficerIndividualDto.getLastName(), errors, loggingContext);
+            boolean hasFormerNamesFlagValid = validateHasFormerNames(managingOfficerIndividualDto.getHasFormerNames(), errors, loggingContext);
```

Comment:
line 33 & 42
I'd start the boolean vars with an 'is'
isHasFormerNamesFlagValid
isSameAddressFlagValid
so it reads , is 'x' valid

Reply:
Updated the names.